packages/prompts/src/index.ts

```diff
@@ -27,6 +27,7 @@ const S_STEP_SUBMIT = s('◇', 'o');
 
 const S_BAR_START = s('┌', 'T');
 const S_BAR = s('│', '|');
+const S_BAR_DETAIL = s('│ⓘ', `|${color.italic('i')}`);
```

Comment:
Flagging this since I want to play with it locally and come up with a good indicator. Having an icon on the inside margin feels a bit off, but I don't know what will feel better just yet.

Thanks for including your explorations in the PR description!